lib/cli/src/helpers.ts

```diff
@@ -344,3 +345,42 @@ export function copyTemplate(templateRoot: string, storyFormat: StoryFormat) {
   }
   fse.copySync(templateDir, '.', { overwrite: true });
 }
+
+export function copyComponents(framework: SupportedFrameworks, language: SupportedLanguage) {
+  const languageFolderMapping: { [key in SupportedLanguage]: string } = {
```

Comment:
`{ [key in SupportedLanguage]: string }` can be replaced by TS `Record` builtin type: `Record<SupportedLanguage, string>`

Reply:
I always forgot about it. Thanks !